### PR TITLE
feat: [APAF-1316] styling, redirect, version control

### PR DESCRIPTION
**Pull request by josie-ku-awx**
feat: [APAF-1316] styling updates to demo (include error hints, clean code)

feat: [APAF-1316] added redirect element (for every demo)

chore: [APAF-1316] add npm package version control 

docs: [APAF-1316] update readme links (clean up links)

[APAF-1316]: https://airwallex.atlassian.net/browse/APAF-1316
[APAF-1316]: https://airwallex.atlassian.net/browse/APAF-1316
[APAF-1316]: https://airwallex.atlassian.net/browse/APAF-1316
[APAF-1316]: https://airwallex.atlassian.net/browse/APAF-1316

### CODE REVIEW

File: integrations/angular/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@angular/platform-browser": "~11.0.0",
     "@angular/platform-browser-dynamic": "~11.0.0",
     "@angular/router": "~11.0.0",
-    "airwallex-payment-elements": "*",
+    "airwallex-payment-elements": "0.2.11",
```
Comment (CharlieLangCL) [inline integrations/angular/package.json:20]: We are propose for latest version, so I think don't need to specific the version

File: docs/redirect.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      // STEP #7: Add an event listener to handle events when the payment is successful.
+      window.addEventListener('onSuccess', (event) => {
+        /*
+          ... Handle event on success
+        */
+        window.alert(event.detail);
+      });
+
+      // STEP #8: Add an event listener to handle events when the payment has failed.
+      window.addEventListener('onError', (event) => {
```
Comment (CharlieLangCL) [inline docs/redirect.md:179]: Not sure if redirect will fire those two event

File: integrations/vue/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "lint": "eslint \"**/*.{js,vue}\" "
   },
   "dependencies": {
-    "airwallex-payment-elements": "*",
+    "airwallex-payment-elements": "0.2.11",
```
Comment (CharlieLangCL) [inline integrations/vue/package.json:14]: same here

File: docs/redirect.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+```html
+<!DOCTYPE html>
+<html>
+  <head lang="en">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Airwallex Checkout Playground</title>
+    <!-- STEP #1: Import airwallex-payment-elements bundle -->
+    <script src="https://checkout.airwallex.com/assets/bundle.0.0.xx.min.js"></script>
```
Comment (CharlieLangCL) [inline docs/redirect.md:138]:
```suggestion
    <script src="https://checkout.airwallex.com/assets/bundle.xx.xx.xx.min.js"></script>
```

File: integrations/angular/package.json (hunk 1 above)
Comment (josie-ku-awx) [reply integrations/angular/package.json:20]: Not sure why it didn't upgrade my package automatically on yarn when I tried it, but I will revert it to an asterisk.

**Review by CharlieLangCL — APPROVED** [review]

**Pull request merged**